astropy/wcs/docstrings.py

```diff
@@ -666,15 +666,11 @@ def RA_DEC_ORDER(indent=0):
 """
 
 extlev = """
-``int`` (read-only)
-
-``EXTLEV`` identifying the binary table extension.
+``int`` (read-only) ``EXTLEV`` identifying the binary table extension.
```

Comment:
What is the reason to remove the blank line separation here?

Reply:
Because with that line, only `int (read-only)` would appear in the table listing attributes of the class. Compare to `Tabprm` - same thing is there - "short description" is on a single line.

Reply:
Because with that line, only `int (read-only)` would appear in the table listing attributes of the class. Compare to `Tabprm` - same thing is there - "short description" is on a single line.

In other words, if `Tabprm` were documented with a line between `int` (or whatever type) and description, then in ['Attributes Summary'](https://docs.astropy.org/en/stable/api/astropy.wcs.Tabprm.html) table, each row would have had only type.